sse2neon.h

```diff
@@ -815,6 +815,88 @@ FORCE_INLINE __m128 _mm_cmpneq_ps(__m128 a, __m128 b)
         vceqq_f32(vreinterpretq_f32_m128(a), vreinterpretq_f32_m128(b))));
 }
 
+// Compares for inequality.
```

Comment:
We can provide `_mm_move_ss` as following:
```c

return vreinterpretq_m128_f32(
    vsetq_lane_f32(vgetq_lane_f32(vreinterpretq_f32_m128 (b), 0),
                               vreinterpretq_f32_m128 (a), 0));
```

And then, we can reimplement several intrinsics using `_mm_move_ss`.

For example:
```c
FORCE_INLINE __m128 _mm_cmpneq_ss(__m128 a, __m128 b)
{
    return _mm_move_ss(a, _mm_cmpneq_ps(a, b));
}
```